test/gtest/ucp/test_ucp_tag_xfer.cc

```diff
@@ -248,7 +248,13 @@ size_t test_ucp_tag_xfer::get_msg_size()
 {
     size_t raw_size   = 1148544 / ucs::test_time_multiplier();
     size_t chunk_size = num_lanes() * UCS_SYS_PCI_MAX_PAYLOAD;
-    return ucs_div_round_up(raw_size, chunk_size) * chunk_size;
+    size_t page_size  = ucs_get_page_size();
+
+    EXPECT_TRUE(ucs_is_pow2(chunk_size) && ucs_is_pow2(page_size)) <<
+                "message size should be aligned to both page and chunk sizes";
+    size_t alignment  = ucs_max(chunk_size, page_size);
```

Comment:
I think chunk_size may be not a power of 2 - if num_lanes is not a power of 2
maybe `alignment = ucs_max(page_size, UCS_SYS_PCI_MAX_PAYLOAD) * num_lanes()`

Reply:
Done.